# Definizione del servizio per controllare lo stato delle lampadine

string command
---
string risposta
